### PR TITLE
Add .rubocop.yml

### PR DESCRIPTION
**Pull request by salbertson**
Hound users will be able to use our style guides after https://github.com/thoughtbot/hound/pull/952 is released.

```
ruby:
  config_file: https://raw.githubusercontent.com/thoughtbot/guides/master/style/ruby/.rubocop.yml
```

Feedback on this PR or https://github.com/thoughtbot/hound/pull/952 would be great.

### CODE REVIEW

**Comment by gabebw** [comment]
Can you add descriptions of each rule? I'm not sure what each one means.

**Comment by jferris** [comment]
This is excellent news! Does this mean changes to `.rubocop.yml` in the guides will affect how Hound comments on our repositories?

**Comment by salbertson** [comment]
@gabebw yes, done.

**Comment by salbertson** [comment]
@jferris that's the plan!

**Comment by jferris** [comment]
> that's the plan!

```
 .----------------.  .----------------.  .----------------.  .----------------.  .----------------.  .----------------. 
| .--------------. || .--------------. || .--------------. || .--------------. || .--------------. || .--------------. |
| |  ____  ____  | || | _____  _____ | || |   ________   | || |   ________   | || |      __      | || |  ____  ____  | |
| | |_   ||   _| | || ||_   _||_   _|| || |  |  __   _|  | || |  |  __   _|  | || |     /  \     | || | |_   ||   _| | |
| |   | |__| |   | || |  | |    | |  | || |  |_/  / /    | || |  |_/  / /    | || |    / /\ \    | || |   | |__| |   | |
| |   |  __  |   | || |  | '    ' |  | || |     .'.' _   | || |     .'.' _   | || |   / ____ \   | || |   |  __  |   | |
| |  _| |  | |_  | || |   \ `--' /   | || |   _/ /__/ |  | || |   _/ /__/ |  | || | _/ /    \ \_ | || |  _| |  | |_  | |
| | |____||____| | || |    `.__.'    | || |  |________|  | || |  |________|  | || ||____|  |____|| || | |____||____| | |
| |              | || |              | || |              | || |              | || |              | || |              | |
| '--------------' || '--------------' || '--------------' || '--------------' || '--------------' || '--------------' |
 '----------------'  '----------------'  '----------------'  '----------------'  '----------------'  '----------------' 
```

**Comment by salbertson** [comment]
@gabebw @jferris anything else?

**Comment by jferris** [comment]
Nope, this looks solid to me.

File: style/ruby/.rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+  Enabled: true
+
+Style/TrailingComma:
+  Description: 'Checks for trailing comma in parameter lists and literals.'
+  StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas'
+  EnforcedStyleForMultiline: comma
+  SupportedStyles:
+    - comma
+    - no_comma
+  Enabled: true
```
Comment (mike-burns) [inline style/ruby/.rubocop.yml]: Both styles are supported; is that the same as `Enabled: false`?

Comment (teoljungberg) [inline style/ruby/.rubocop.yml]: I think these are the choices you can make in `EnforceStyleForMultiline`.

**Pull request merged**